src/app/notes/notes.component.scss

```diff
@@ -0,0 +1 @@
+@import "src/app/app.component.scss";
```

Comment:
this is not needed here (nor into other components) if there style are in app.component. if it doesn't work that way just import in into styleUrls in the `component.ts` file, but as i said, `@imports` into component styles are only for importing styles from 3rd party libraries, not our own styles